english/dream/lucid.md

```diff
@@ -0,0 +1,8 @@
+You’ve had some experience with lucid dreaming and begin to manipulate the dream so you have an escape from this
+marshmallow tomb. You intensely focus on the marshmallow walls around you. The wall in front of you begins to vibrate
+and slowly begins to separate. Through the opening in the wall you see a cobblestone path leading towards a castle
+in the distance.
+
+Do you:
+
+[Pinch yourself] (create-your-own-adventure/english/pinch/pinch.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.